commcare_connect/opportunity/tables.py

```diff
@@ -147,15 +153,24 @@ class Meta:
         empty_text = "No users invited for this opportunity."
         orderable = False
 
+    def render_display_name(self, record):
+        if record.opportunity_access is None:
```

Comment:
This is obviously a different PR, but we may need to be careful about this when the credential workflow rolls out because this would expose the phone number of anyone with a given credential (which may not be what we want to do before they accept)

Reply:
Thanks for pointing this out. I have a query regarding this. Can we show phone numbers for users who don't have a ConnectId account?

Reply:
Added. https://github.com/dimagi/commcare-connect/pull/305/commits/fe9a83e79dfe6161c811470f0781185ebb03c2ed

I updated the code to only show phone numbers for users who have not registered on ConnectID.